security-analysis/security-analysis-api/src/test/resources/SecurityAnalysisResult.json

```diff
@@ -25,6 +25,14 @@
       "actionsTaken" : [ ]
```

Comment:
Actually, we need to increment the version to 1.2, because version 1.1 is not supposed to contain "operatorStrategyResults".

Also, in 1.2, we should be consistent by moving "branchResults" etc. into a new "networkResult" object